R/function-filtering.R

```diff
@@ -0,0 +1,142 @@
+#' @title Basic filtering functions for metabolomics
+#'
+#' @description
+#'
+#' When dealing with metabolomics results, it is often necessary to filter
+#' features based on certain criteria. These criteria are typically derived
+#' from statistical formulas applied to full rows of data, where each row
+#' represents a feature. The following functions provide basic filtering
```

Comment:
also here then maybe better to talk about quality assessment instead. e.g. "The following functions allow to calculate basic quality assessment estimates commonly used..."